app-arch/lz4/files/1.10.0-fix-freestanding-test.patch

```diff
@@ -0,0 +1,26 @@
+
```

Comment:
Could you `cat` both patches into a single file to save inodes?

Reply:
Done.